<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AVBaseWidgetClass</class>
 <widget class="QWidget" name="AVBaseWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1714</width>
    <height>972</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AVBaseWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="CustomLabel" name="labelFileList">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>音频文件列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FilePathIconListWidget" name="audioFileList">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="AVFrame">
        <property name="minimumSize">
         <size>
          <width>1080</width>
          <height>720</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1080</width>
          <height>720</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="controlLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="CustomProgressBar" name="customProgressBar">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="progressBarStyle" stdset="0">
           <enum>CustomProgressBar::ProgressBarStyle_Rounded</enum>
          </property>
          <property name="enableGradient" stdset="0">
           <bool>true</bool>
          </property>
          <property name="enableAnimation" stdset="0">
           <bool>true</bool>
          </property>
          <property name="animationDuration" stdset="0">
           <number>300</number>
          </property>
          <property name="textPosition" stdset="0">
           <enum>CustomProgressBar::TextPosition_Center</enum>
          </property>
          <property name="enableShadow" stdset="0">
           <bool>true</bool>
          </property>
          <property name="enableBorder" stdset="0">
           <bool>true</bool>
          </property>
          <property name="borderWidth" stdset="0">
           <number>1</number>
          </property>
          <property name="borderRadius" stdset="0">
           <number>8</number>
          </property>
          <property name="enableStripes" stdset="0">
           <bool>false</bool>
          </property>
          <property name="stripeDirection" stdset="0">
           <enum>CustomProgressBar::StripeDirection_LeftToRight</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ControlButtonWidget" name="ControlButtons" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="RightLayoutFrame">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="ToolButtonFrame">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CoreWidget/CustomLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomProgressBar</class>
   <extends>QProgressBar</extends>
   <header>CoreWidget/CustomProgressBar.h</header>
  </customwidget>
  <customwidget>
   <class>FilePathIconListWidget</class>
   <extends>QListWidget</extends>
   <header>DomainWidget/FilePathIconListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ControlButtonWidget</class>
   <extends>QWidget</extends>
   <header>ControlButtonWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
